data_pipelines_cli/cli_commands/compile.py

```diff
@@ -146,7 +151,13 @@ def compile_project(
 @click.option(
     "--docker-args", type=str, required=False, help="Args required to build project in json format"
 )
+@click.option(
+    "--bi-build",
```

Comment:
I'm thinking about turning it on and off from bi.yml instead. Maybe both could be used. We would like to control it without changes in CICD process :)